src/modules/shared/translations/en-us.yml

```diff
@@ -25,3 +26,7 @@ parts:
       title: "Error state go to the homepage link"
       screenshot: "https://drive.google.com/file/d/1rJ9c4mW0YWnL5d6KWWZaY9TJBEH5otLL/view?usp=drive_link"
       value: "Go to the homepage"
+  - translation:
```

Comment:
Is the addition of this translation the reason why we are removing the close label?

Reply:
Yes, before we were reading the `closeLabel` from Curlybars using the `t` helper: https://github.com/zendesk/copenhagen_theme/blob/a808e3c7e9b6614c2364f9172245923523461f9f/templates/document_head.hbs#L29 and we were passing it to the JS module.
This was done just because we had only 1 string to load in the `flash-notifications` module and seemed overkill to create a JSON translation file with just that string

Now we have a translation file for the shared module with this string, and we can fetch it like the other strings

Reply:
Thank you for explaining. I think this answers my other comment as well.